README.md

```diff
@@ -1,3 +1,53 @@
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
 # school-library
 Implement classes, objects, encapsulation and inheritance with Ruby.
 Module 4 - Week 3 - Microverse - Full-Stack Web Development
+
+## Built With
+
+- Ruby
+
+## Getting Started
+
+### To launch the project locally:
+
+#### Step 1:
+
+- Clone this repo locally using git in the command line and the following command
+
+ ```bash
+ > git clone git@github.com:fernando-silvabr66/school-library.git
+ ```
+
+- Alternatively, you can just download the complete zip file and extract the folder in your directory
+
+#### Step 2:
+
+In the project directory, you can run:
+
+```bash
+#### > ruby <<< to be defined.rb >>> 
+```
+
+or alternatively you can copy and paste the commands in
+
+```bash
+> irb 
+```
+## Author
+
+👤 **Fernando César da Silva**
+
+- GitHub: [@fernando-silvabr66](https://github.com/fernando-silvabr66)
+- Twitter: [@fsilvabr1](https://twitter.com/fsilvabr1)
+- LinkedIn: [https://linkedin.com/in/fernando-silvabr66](https://linkedin.com/in/fernando-silvabr66)
+
+
+## 🤝 Contributing
+
+Give a ⭐️ if you like this project!
+
+## 📝 License
+
+This project is [MIT](./LICENSE.md) licensed.
```

Comment:
- [ ] Kindly add the `Acknowledgement` section in the Readme and fix the MIT License broken link to make your code more informative and professional.

Reply:
Thanks for the suggestions.